packages/fxa-react/components/LoadingSpinner/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { ReactComponent as Spinner } from './spinner.svg';
+
+export const LoadingSpinner = ({
+  className = '',
```

Comment:
Unless we are concatenating class names like ```className={`${svgClassName} spinner-image`}```, you should be able to leave `className` without a default. When React components receive `undefined`, they don't render that property. So I think this is going to get you `<div class="" data-testid="loading-spinner">` as opposed to if it's left `undefined`, it should only render `<div data-testid="loading-spinner">`.